quantumlib/Qualtran
# Sync up check and dev_tools scripts with cirq

- Remove check/pytest option --actually-quiet
- Remove unused variable `CIRQ_TESTING_RANDOM_SEED`
- Remove unused special-casing for Python 3.6
- check/pytest - simplify handling of the parallel job count
  `-n COUNT` option.  Respect both the `-n0` and `-n 0` option formats.
- Fix issues reported by shellcheck.
- Clean up cirq leftovers from the `check` and `dev_tools` scripts
